build.gradle.kts

```diff
@@ -86,6 +87,15 @@ sonarqube {
     }
 }
 
+conventionalCommits {
+    scopes = subprojects.map { it.name }.toList()
+    failureMessage = "Error : The commit message does not meet the Conventional Commit standard\n" +
+            "An example of a valid message is:\n" +
+            "feat(buttons): add the small size button\n" +
+            "Available scopes : ${scopes}\n" +
+            "More details at: https://www.conventionalcommits.org/en/v1.0.0/#summary"
+}
```

Comment:
I see that you are using the `scopes`. 
Is there any particular reason for not using the `types`?

Reply:
We do are using the types, I just didn't listed in the error message as they are the conventional ones listed in https://www.conventionalcommits.org/en/v1.0.0/#summary